build/ci/backend/publish.sh

```diff
@@ -17,4 +17,87 @@
 # GNU General Public License for more details.
 #
 # You should have received a copy of the GNU General Public License
-# along with this program.  If not, see <https://www.gnu.org/licenses/>.
\ No newline at end of file
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+S3_KEY=""
+S3_SECRET=""
+S3_URL="s3://convertor.musescore.org"
```

Comment:
here too

Reply:
I'll rename these later, it's more difficult with it